### PR TITLE
[AA-1099] - Update automapper and log4net

### PR DESCRIPTION
**Pull request by CSR2017**
@saa14 @plioi 
Please help review and merge the changes.
Changes include 
1. Automapper package version update and related code changes.
2. Log4net package version update
Run all the unit test and did application testing.

Did not include the "HtmlTag" changes. Since there are lot of dependency packages getting updated and application pages are throwing error. Will be raising separate PR for that.
Thanks

### CODE REVIEW

**Comment by plioi** [comment]
Let's sync up on the HtmlTags package. There's a trick we might be able to use to minimize the need to deal with its many dependencies.

**Comment by plioi** [comment]
@saa14 When you review this, note some of the AutoMapper changes are simply the recommended upgrade path:

https://docs.automapper.org/en/stable/8.0-Upgrade-Guide.html#resolveusing
https://docs.automapper.org/en/stable/9.0-Upgrade-Guide.html#the-static-api-was-removed

These appear to be the primary changes, though you probably want to scan all the nearby upgrade guide pages from those documents.

File: Application/EdFi.Ods.AdminApp.Management/Api/Automapper/EducationOrganizationAddressResolver.cs [hunk 1]
```diff
@@ -58,7 +58,7 @@ public static class EducationOrganizationAddressResolver
             return addressList;
         }
 
-        public static List<EdFiEducationOrganizationAddress> Resolve(EducationOrganization source)
```
Comment (plioi) [inline Application/EdFi.Ods.AdminApp.Management/Api/Automapper/EducationOrganizationAddressResolver.cs:61]: Are the changes to this file necessary?

File: Application/EdFi.Ods.AdminApp.Management/Api/Automapper/EducationOrganizationAddressResolver.cs [hunk 2]
```diff
@@ -44,7 +44,7 @@ public static class EducationOrganizationAddressResolver
             var addressList = originalList?.ToList() ?? new List<EdFiEducationOrganizationAddress>();
             originalList?.Clear();
 
-            var address = Resolve(source)[0];
```
Comment (plioi) [inline Application/EdFi.Ods.AdminApp.Management/Api/Automapper/EducationOrganizationAddressResolver.cs:47]: I'm nervous about the null here. Is this a side effect of the *old* code in the mapping using ResolveUseing(classname.staticmethodname)?

**Review by saa14 — APPROVED** [review]

**Pull request merged**